remote-scripts/stor_performance_fio.sh

```diff
@@ -12,81 +12,89 @@
 ##
 ###############################################################################
 
-
 dos2unix utils.sh
 
-
 # Source utils.sh
 . utils.sh || {
-    echo "Error: unable to source utils.sh!"
-    exit 1
+	echo "Error: unable to source utils.sh!"
+	exit 1
 }
 
-
 # Source constants.sh to get all paramters from XML <testParams>
 . constants.sh || {
-    echo "Error: unable to source constants.sh!"
-    exit 1
+	echo "Error: unable to source constants.sh!"
+	exit 1
 }
 
-
 # Source constants file and initialize most common variables
 UtilsInit
 
-
 ###############################################################################
 ##
 ## Main
 ##
 ###############################################################################
 #Check the guest version which use python2.x, so install pip
+#Below moudle yaml,click pandas numpy scipy is used by python script RunFioTest.py and GenerateTestReport.py.
 if [[ $DISTRO != "redhat_8" ]]; then
-    yum install -y python-yaml
-    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
-    python get-pip.py
-    pip install click pandas numpy scipy
-    UpdateSummary "For rhel6 and rhel7, python2.x use pip"
+	yum install -y python-yaml
```

Comment:
I found this one can be install with `pip` the name should be `PyYAML`, you can check with `pip freeze`.